docs/080-breaking-changes.rst

```diff
@@ -155,3 +156,15 @@ Interface Changes
   Use the "compact AST" (``--ast-compact--json`` resp. ``AST``) as replacement.
 
 * The old error reporter (``--old-reporter``) has been removed.
+
+
+How to update your code
+=======================
+
+- If you rely on wrapping arithmetic, surround each operation with ``unchecked { ... }``.
+- Optional: If you use SafeMath or a similar library, change ``x.add(y)`` to ``x + y``, ``x.mul(y)`` to ``x * y`` etc.
+- Add ``pragma abicoder v1;`` if you want to stay with the old ABI coder.
+- Change ``byte`` to ``bytes1``.
+- Add intermediate explicit type conversions if required.
+- Combine ``c.f{gas: 10000}{value: 1}()`` to ``c.f{gas: 10000, value: 1}()``.
+- Change ``msg.sender.transfer(x)`` to ``payable(msg.sender).transfer(x)`` or use a stored variable of ``address payable`` type.
```

Comment:
`stored variable` sounds a bit too close to `storage variable` in my opinion and could be confusing because of that:
```suggestion
- Change ``msg.sender.transfer(x)`` to ``payable(msg.sender).transfer(x)`` or use an intermediate variable of ``address payable`` type.
```

Reply:
I really meant 'stored variable'. The reason is that you often compare `msg.sender` with `owner`, and then you can just make `owner` payable.

Reply:
Oh. Ok then. But it would work just as well in a local variable, right?